lib/guides_style_18f/includes/header.html

```diff
@@ -3,6 +3,51 @@
   <meta charset="utf-8">
   <title>{{ page.title }} - {{ site.name }}</title>
   <meta name="viewport" content="width=device-width">
+
+
+
+{% if site.url %}
```

Comment:
I'd only fence the tags that depend on site.url...no reason to include the others.